aggregator/src/test/java/org/jenkinsci/plugins/workflow/CpsScmFlowDefinitionTest.java

```diff
@@ -106,6 +107,23 @@
         assertFalse(iterator.hasNext());
     }
 
+    @Test public void emptyChangeLogEmptyChangeSets() throws Exception {
+        sampleRepo.init();
+        sampleRepo.write("flow.groovy", "echo 'version one'");
+        sampleRepo.git("add", "flow.groovy");
+        sampleRepo.git("commit", "--message=init");
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        p.addTrigger(new SCMTrigger("")); // no schedule, use notifyCommit only
```

Comment:
:ant: could be omitted, since we are not using webhook notification here anyway.